fhirpipe/run.py

```diff
@@ -52,8 +54,17 @@ def run(
     transformer = Transformer(pool)
     loader = Loader(fhirstore, bypass_validation, pool)
 
+    # TODO is there a more elegant way to skip_ref_binding?
+    binder = ReferenceBinder(fhirstore, skip_ref_binding)
```

Comment:
I would not pass the option to the constructor and call binder.add_references only if `skip_ref_binding` is False